README.md

```diff
@@ -21,7 +21,8 @@ and add to your esformatter config file:
   ],
   "quotes": {
     "type": "single",
-    "avoidEscape": false
+    "avoidEscape": false,
+    "normalizeObjectKeys": false
```

Comment:
This is inconsistent with the example provided in #1. `keys` is probably shorter and as easy to understand (it is implied that `esformatter-plugins` will normalize).